package.json

```
@@ -48,21 +48,21 @@
   "module": "rites/roman1969/dist/esm/romcal.js",
   "typings": "rites/roman1969/dist/index.d.ts",
   "scripts": {
-    "build": "npm run build -w=@romcal/easter && npm run build -w=@romcal/rite-roman1969",
-    "clean": "rimraf coverage dist tmp && npm run clean -w=@romcal/easter && npm run clean -w=@romcal/rite-roman1969",
-    "data:checks": "npm run data:checks -w=@romcal/rite-roman1969",
-    "doc": "npm run doc -w=@romcal/rite-roman1969",
-    "docs:check-links": "npm run docs:check-links -w=@romcal/rite-roman1969",
-    "docs:sort-glossary": "npm run docs:sort-glossary -w=@romcal/rite-roman1969",
-    "lint": "npm run lint -w=@romcal/easter && npm run lint -w=@romcal/rite-roman1969",
-    "lint:fix": "npm run lint -w=@romcal/easter && npm run lint:fix -w=@romcal/rite-roman1969",
+    "build": "npm run build -w=@internal/easter && npm run build -w=@internal/rite-roman1969",
```

if we're building or cleaning on multiple, they can all be listed, e.g.: `npm run test --workspace=a --workspace=b` ([ref](https://docs.npmjs.com/cli/v10/using-npm/workspaces))

```suggestion
    "build": "npm run build -w=@internal/easter -w=@internal/rite-roman1969",
```